code/crates/discovery/src/config.rs

```diff
@@ -5,9 +5,9 @@ const DEFAULT_NUM_INBOUND_PEERS: usize = 20;
 
 const DEFAULT_EPHEMERAL_CONNECTION_TIMEOUT: Duration = Duration::from_secs(15);
 
-const DEFAULT_DIAL_MAX_RETRIES: usize = 5;
-const DEFAULT_PEERS_REQUEST_MAX_RETRIES: usize = 5;
-const DEFAULT_CONNECT_REQUEST_MAX_RETRIES: usize = 0;
+const DEFAULT_DIAL_MAX_RETRIES: usize = 10_000;
+const DEFAULT_PEERS_REQUEST_MAX_RETRIES: usize = 10_000;
+const DEFAULT_CONNECT_REQUEST_MAX_RETRIES: usize = 10_000;
```

Comment:
These changes should not be needed if max_retries is set sufficiently high in the config.

Reply:
Which config are you referring to? These parameters cannot be set explicitly yet; the default values are used.